src/content/reference/react/StrictMode.md

```diff
@@ -122,6 +122,12 @@ function App() {
 
 In this example, Strict Mode checks will not run against the `Header` and `Footer` components. However, they will run on `Sidebar` and `Content`, as well as all of the components inside them, no matter how deep.
 
+<Note>
+
+When `StrictMode` is enabled for a part of the app, React will only enable behaviors that are possible in production. For example, if `<StrictMode>` is not enabled at the root of the app, it will not [re-run Effects an extra time](#fixing-bugs-found-by-cleaning-up-and-re-attaching-dom-refs-in-development) on initial mount, since this would cause child effects to double fire without the parent effects, which cannot happen in production.
```

Comment:
Hey, @rickhanlonii. Just realized this link to `#fixing-bugs-found-by-cleaning-up-and-re-attaching-dom-refs-in-development` is broken.

Reply:
Thanks, fixed here https://github.com/reactjs/react.dev/pull/7713!